packages/expo-image-manipulator/android/src/main/java/expo/modules/imagemanipulator/ImageManipulatorModule.java

```diff
@@ -69,25 +68,15 @@ public void manipulateAsync(final String uri, final ArrayList<Object> actions, f
       return;
     }
 
-
-    mImageLoader.loadImageFromURL(uri, new ImageLoader.ResultListener() {
-      @Override
-      public void onSuccess(@NonNull Bitmap bitmap) {
-        processBitmapWithActions(bitmap, manipulatorActions, manipulatorSaveOptions, promise);
-      }
-
-      @Override
-      public void onFailure(@Nullable Throwable cause) {
-        // No cleanup required here.
-        String basicMessage = "Could not get decoded bitmap of " + uri;
-        if (cause != null) {
-          promise.reject(ERROR_TAG + "_DECODE",
-              basicMessage + ": " + cause.toString(), cause);
-        } else {
-          promise.reject(ERROR_TAG + "_DECODE", basicMessage + ".");
-        }
-      }
-    });
+    Glide.with(getContext())
+        .asBitmap()
+        .load(uri)
+        .into(new SimpleTarget<Bitmap>() {
```

Comment:
Will this while invocation cache the image? Do we want to cache the images with ImageManipulator?